src/gamedata/g_mapinfo.cpp

```diff
@@ -1644,6 +1644,7 @@ MapFlagHandlers[] =
 	{ "nolightfade",					MITYPE_SETFLAG3,	LEVEL3_NOLIGHTFADE, 0 },
 	{ "nocoloredspritelighting",		MITYPE_SETFLAG3,	LEVEL3_NOCOLOREDSPRITELIGHTING, 0 },
 	{ "forceworldpanning",				MITYPE_SETFLAG3,	LEVEL3_FORCEWORLDPANNING, 0 },
+	{ "propermonsterdamage",			MITYPE_SETFLAG3,	LEVEL3_PROPERMONSTERDAMAGE, 0 },
```

Comment:
I don't like the name given here. Shouldn't it be "monsterfallingdamage" or something like that?